libbeat/tests/integration/framework.go

```diff
@@ -805,12 +842,12 @@ func GetESURL(t *testing.T, scheme string) url.URL {
 		}
 	}
 
-	user := os.Getenv("ES_USER")
+	user := os.Getenv("ES_SUPERUSER_USER")
```

Comment:
The mage automation sets those environment variables:
https://github.com/elastic/beats/blob/ba454d5d6b35c1db9e4b78668908c561f228116d/dev-tools/mage/integtest_docker.go#L178-L198

And the previous one (`ES_USER`/`ES_PASS`) didn't have enough permission to run some of the tests.